src/content/certifications/exam-faqs/vault-associate-003.mdx

```diff
@@ -0,0 +1,123 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
+
+## Exam Availability
+Release date January 6 2025
```

Comment:
```suggestion
Coming soon!
```

Reply:
I would suggest not stating the date and instead just leaving it as "coming soon"